tests/use-cases/tag-use-case.test.ts

```diff
@@ -117,7 +117,7 @@ describe('The tag use case with SHA', () => {
     when(
       pullRequestCommitExtractorMock.commits(anything(), 'repository')
     ).thenReturn(of(['A commit message']));
-    when(shaCommitExtractorMock.commits('sha', 'repository')).thenReturn(
+    when(shaCommitExtractorMock.commits(anything(), 'repository')).thenReturn(
```

Comment:
For some reason putting the actual value here and below doesn't work :thinking: 
So I switched back to `anything()` which works.
And obviously I tested it out with the custom Jira/Git repos and everything worked :ok_hand:

Reply:
You need the deepEqual function.
If you look some other tests you can see the usage.